repos/keg-cli/src/tasks/docker/tunnel.js

```diff
@@ -0,0 +1,54 @@
+const ngrok = require('ngrok')
+const qrcode = require('qrcode-terminal')
+const { Logger } = require('KegLog')
+
+/**
+ * Create a public tunnel the the url (by default, to `kegdev.xyz`)
+ * @function
+ * @param {Object} args - arguments passed from the runTask method
+ * @param {string} args.command - Initial command being run
+ * @param {Array} args.params - arguments passed from the command line
+ *
+ * @returns {void}
+ */
+const tunnel = async args => {
+  const { params } = args
+  const { qr, source, port } = params
+
+  const ip = process.env.KEG_DOCKER_IP
```

Comment:
While this is not wrong, there's a better way to do this. 
```js
// import the docker constants at the top of the file
const { DOCKER } = require('KegConst/docker')

// Then on this line set it like this
const ip = DOCKER.MACHINE.IP
// Or like this, because at some point we'll switch to docker for desktop
const ip = DOCKER.PREFIXED.KEG_DOCKER_IP
```
Most likely when we switch to docker for desktop, this will have to be updated anyways, so either option is fine

Reply:
I was wondering if something like that existed. Thanks, just updated